signac/contrib/job.py

```diff
@@ -22,14 +23,15 @@
 
 class _sp_save_hook(object):
 
-    def __init__(self, job):
-        self.job = job
+    def __init__(self, *jobs):
+        self.jobs = list(jobs)
```

Comment:
Does this change belong in this branch? It seems like a better fit for the grouping PR.

Reply:
Ah I see, this is necessary for enabling pickling? Because of the way that the state is being tracked below?

Reply:
No, this change is necessary, because we need to be able to add multiple jobs to the same state point hook. A shallow copy of a job will share the same hook.